MDANSE_GUI/Src/MDANSE_GUI/InputWidgets/TrajectoryFilterWidget.py

```diff
@@ -1027,15 +1092,65 @@ def resample_and_normalise(self, values, to_len):
         """
         return signal.resample(values, to_len) / values.max()
 
+    @Slot(object)
+    def accept_results(self, res_dict: dict[str, np.ndarray]):
```

Comment:
```suggestion
    def accept_results(self, res_dict: dict[str, npt.NDArray[float]]):
```
Gives better typing information to type checkers.